epoch/api/channels_api_usage.md

```diff
@@ -0,0 +1,462 @@
+[back](./README.md)
+# Channels - intended usage
```

Comment:
This document seems a mix of WS schema - that other sets of WS APIs define as [separate pages](https://github.com/aeternity/protocol/blob/5efa70c54fe0de186239892bd69e51e385e7371e/epoch/api/README.md#list-of-ws-apis) - and intended usage.

Reply:
The channels' API is a combination of websocket (open, mutual close, udpates) and HTTP (deposit, withdrawal and solo close related)  requests and it makes little sense of splitting those into different pages.
Probably it is misleading because the HTTP requests are not yet described (since they are not yet fully implemented...)

Reply:
Tracked as https://www.pivotaltracker.com/story/show/157476849 - to be validated. Not required for this PR.